runtime.go

```diff
@@ -3229,3 +3236,11 @@ func (r *Runtime) getPrototypeFromCtor(newTarget, defCtor, defProto *Object) *Ob
 	}
 	return defProto
 }
+
+func (self *Runtime) DisableFunctionTickTracking() {
+	self.disableTickMetrics = true
+}
+
+func (self *Runtime) EnableFunctionTickTracking() {
+	self.disableTickMetrics = false
+}
```

Comment:
This method might not be necessary since we likely won't flip metrics on/off during a function run.